### PR TITLE
Response added in the OnHold dropdown

### PR DESCRIPTION
**Pull request by divyav-aot**

### CODE REVIEW

File: forms-flow-web/src/components/FOI/customComponents/Attachments/AttachmentModal.js [hunk 1]
```diff
@@ -1,46 +1,54 @@
-import React, { useEffect, useState } from 'react';
+import React, { useEffect, useState } from "react";
```
Comment (divyav-aot) [inline forms-flow-web/src/components/FOI/customComponents/Attachments/AttachmentModal.js:1]: Resolved conflicts for AttachmentModal.js

File: forms-flow-web/src/components/FOI/customComponents/StateDropDown.js [hunk 2]
```diff
@@ -94,37 +98,37 @@ const StateDropDown = ({
...
       _stateList = MinistryStateList;
     }
-    const personalRequest = requestType?.toLowerCase() === FOI_COMPONENT_CONSTANTS.REQUEST_TYPE_PERSONAL;
+    const personalRequest =
+      requestType?.toLowerCase() ===
+      FOI_COMPONENT_CONSTANTS.REQUEST_TYPE_PERSONAL;
     const personalIAO = !_isMinistryCoordinator && personalRequest;
-
+    const previousState =
+      stateTransition?.length > 0 && stateTransition[1]?.status;
```
Comment (divyav-aot) [inline forms-flow-web/src/components/FOI/customComponents/StateDropDown.js:106]: Moved "previousState" to the top

File: forms-flow-web/src/components/FOI/customComponents/StateDropDown.js [hunk 3]
```diff
@@ -136,50 +140,69 @@ const StateDropDown = ({
...
+              _state.status.toLowerCase() !==
+              StateEnum.callforrecords.name.toLowerCase()
+          );
+        } else if (previousState === StateEnum.feeassessed.name) {
+          return _stateList.onhold.filter(
+            (_state) =>
+              _state.status.toLowerCase() !==
+              StateEnum.response.name.toLowerCase()
+          );
+        }
```
Comment (divyav-aot) [inline forms-flow-web/src/components/FOI/customComponents/StateDropDown.js:175]: Updated the onhold statelist based on previousState condition. If previousState == Response then include response else include CFR.

File: forms-flow-web/src/constants/FOI/statusEnum.js [hunk 4]
```diff
@@ -1,74 +1,214 @@
...
+  ],
+  feeassessedforpersonal: [
+    { status: "Fee Estimate", isSelected: false },
+    { status: "Call For Records", isSelected: false },
+    { status: "Closed", isSelected: false },
+  ],
+  onhold: [
+    { status: "On Hold", isSelected: false },
+    { status: "Call For Records", isSelected: false },
+    { status: "Response", isSelected: false },
```
Comment (divyav-aot) [inline forms-flow-web/src/constants/FOI/statusEnum.js:73]: Added Response in the onhold list

**Review by aparna-aot — APPROVED** [review]

**Pull request merged**